internal/service/privatelinkendpointservice/data_source_privatelink_endpoint_service.go

```diff
@@ -85,10 +85,6 @@ func DataSource() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-						"service_attachment_name": {
-							Type:     schema.TypeString,
-							Computed: true,
-						},
```

Comment:
`service_attachment_name` was deprecated in the resource but by mistake was not deprecated in the data source. Given that it is not being populated and is being removed in resource, would opt to remove here as well for consistency.